README.md

```diff
@@ -92,15 +92,23 @@ Setup
 Note for **macOS**: If you installed weechat with Homebrew, you will have to locate the python runtime environment used.
 If `--with-python@2` was used, you should use: `sudo /usr/local/opt/python@2/bin/pip2 install websocket_client`
 
-### 2. Download wee\_slack.py to ~/.weechat/python
+### 2. Download wee\_slack.py
 
-If you don't want wee\_slack to start automatically when weechat starts, you can skip the last command.
+In order to [load this Python script in WeeChat](https://weechat.org/files/doc/stable/weechat_scripting.en.html#load_script), it should be located under the `python/` directory in [the WeeChat data files directory](https://weechat.org/files/doc/stable/weechat_user.en.html#files_and_directories), `$XDG_DATA_HOME/weechat/python`, or `$HOME/.local/share/weechat/python/` if `$XDG_DATA_HOME` is not defined or is empty. (These directories will be created the first time you run WeeChat, or you can create them yourself with the first command below.) Download the script to that location:
 
 ```
-mkdir -p ~/.weechat/python/autoload
-cd ~/.weechat/python
+mkdir -p $XDG_DATA_HOME/weechat/python/autoload/
+cd $XDG_DATA_HOME/weechat/python/
```

Comment:
I think I'll change this to `.local/share/weechat` though, as I think most systems don't have `$XDG_DATA_HOME` set (and thus uses `.local/share/weechat`), and I want these commands to work verbatim for most people.

Could use a variable with a default value, but that's extra syntax not everyone will understand, plus it won't work in fish and other shells with different syntax from bash, so I don't think I'll do that.